.gitignore

```
@@ -5,7 +5,6 @@ project.properties
 .settings
 bin/
 gen/
-gplay/
 .idea/
```

excluding the top-level gplay directory is still needed once one is doing a release, this directory is created and get filled with artefacts

however, it should be `/gplay/` to really only exclude the top-level directory and not `/src/gplay/`

```suggestion
/gplay/
.idea/
```